nvme_gw_cli.py

```diff
@@ -0,0 +1,263 @@
+        self._logger = nvme_config.logger
+
+    @cli.cmd([
+        argument("-r", "--rbd", help="RBD image name", required=True),
+        argument("-p", "--pool", help="Ceph pool name", required=True),
+        argument("-n", "--name", help="Bdev name"),
+        argument("-u", "--user", help="User ID"),
+        argument("-b", "--block_size", help="Block size", default=4096),
+    ])
+    def create_bdev(self, args):
+        """
+        Creates a bdev from a Ceph RBD.
+        """
+        try:
+            create_req = pb2.bdev_create_req(
+                ceph_pool_name=args.pool,
+                rbd_name=args.rbd,
+                block_size=int(args.block_size),
+            )
+            ret = self.stub.bdev_rbd_create(create_req)
+            self.logger.info(f"Created bdev: {ret.bdev_name}")
+        except Exception as error:
+            self.logger.error(f"Failed to create bdev: \n {error}")
+
+    @cli.cmd([
+        argument("-n", "--subnqn", help="Subsystem NQN", required=True),
+        argument("-s", "--serial", help="Serial number", required=True),
+    ])
+    def create_subsystem(self, args):
+        """
+        Creates a new subsystem.
+        """
+        try:
+            create_req = pb2.subsystem_create_req(subsystem_nqn=args.subnqn,
+                                                  serial_number=args.serial)
+            ret = self.stub.nvmf_create_subsystem(create_req)
+            self.logger.info(f"Created subsystem: {ret.subsystem_nqn}")
+        except Exception as error:
+            self.logger.error(f"Failed to create subsystem: \n {error}")
+
+    @cli.cmd([
+        argument("-n", "--subnqn", help="Subsystem NQN", required=True),
+        argument("-b", "--bdev", help="Bdev name", required=True),
+    ])
+    def create_namespace(self, args):
+        """
+        Adds a namespace to a previously created subsystem.
+        """
+        try:
+            create_req = pb2.subsystem_add_ns_req(subsystem_nqn=args.subnqn,
+                                                  bdev_name=args.bdev)
+            ret = self.stub.nvmf_subsystem_add_ns(create_req)
+            self.logger.info(f"Added namespace {ret.nsid} to {args.subnqn}")
+        except Exception as error:
+            self.logger.error(f"Failed to add namespace: \n {error}")
+
+    @cli.cmd([argument("-n", "--subnqn", help="Subsystem NQN", required=True)])
+    def allow_any_hosts(self, args):
+        """
+        Allows any host to access a subsystem.
+        """
+        try:
+            create_req = pb2.subsystem_allow_any_host_req(
+                subsystem_nqn=args.subnqn, disable=0)
+            ret = self.stub.nvmf_subsystem_allow_any_host(create_req)
+            self.logger.info(f"All host access to {args.subnqn}: {ret.status}")
+        except Exception as error:
+            self.logger.error(f"Failed to allow any host: \n {error}")
+
+    @cli.cmd([argument("-t", "--type", help="Transport type", default="TCP")])
```

Comment:
```suggestion
    @cli.cmd([argument("-t", "--trtype", help="Transport type", default="TCP")])
```
for consistency with `create_listener`